frontend/public/components/command-line-tools.tsx

```diff
@@ -50,7 +50,7 @@ const CommandLineTools: React.FC<CommandLineToolsProps> = ({obj}) => {
       </p>
       <hr />
       <h2 className="co-section-heading">odo - Developer-focused CLI for OpenShift</h2>
-      <p><TechPreviewBadge /></p>
+      <p><ReleaseBadge releaseBadgeType="tech-preview" /></p>
```

Comment:
No reason to change this from `TechPreviewBadge`

Reply:
Yes, and for that you need to export both `DevPreviewBadge` and `TechPreviewBadge` in the index file of `@console/shared` as suggested above.